pep-0517.txt

```diff
@@ -220,9 +220,45 @@ including any unrecognized files it created.
 
 Mandatory.
 
+::
+
+  def build_sdist(sdist_directory, config_settings):
+      ...
+
+Must build an unpacked source distribution in the specified
```

Comment:
With the sdist now being unpacked, a better name for the hook is likely to be `export_sdist`.

It's also not entirely clear if the sdist contents should be exported directly into the supplied `sdist_directory`, or if the backend should be creating an appropriately named subdirectory.

My inclination would be to leave the directory naming up to the frontend, and have the backend just dump the sdist contents into whichever path is given (as happens with the wheel metadata export)